infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -93,6 +99,10 @@
     // the files of the old instance
     private LogMetadata logMetadata;
 
+    final long logSizeLimit;
+
+    // Resource quota to track the log size
+    ResourceQuota logSizeQuota;
```

Comment:
private final

Reply:
can't make it final, because reset re-assigns it. We need to clean with other class fields later.